subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/BaseGradleImplDepsIntegrationTest.groovy

```diff
@@ -88,36 +88,30 @@ abstract class BaseGradleImplDepsIntegrationTest extends AbstractIntegrationSpec
         """
     }
 
-    static String testablePluginProject(String appliedLanguagePlugin = applyGroovyPlugin()) {
+    static String testableProject(List<String> plugins = ['groovy']) {
```

Comment:
I think you should be able to combine these back together by applying either `java-gradle-plugin` or `groovy-gradle-plugin`.  I think these tests predate the `xxx-gradle-plugin` plugins.

We could also remove the explicit use of gradleApi and test kit here because those are automatically added by the plugin.